src/features/landing/components/CardItem/CardItem.tsx

```diff
@@ -0,0 +1,68 @@
+'use client';
+
+import { Grid, Typography } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface CardItemInterface {
+  item: {
+    id: string;
+    imageSrc: string;
+    title: string;
+    text: string;
+  };
+  type: 'WhyUs' | 'Possibilities';
+}
+
+function CardItem({ item, type }: CardItemInterface) {
```

Comment:
this component can be used across all application, move it to src/components

Reply:
Done